jobs/payment-jobs/tests/jobs/test_cfs_create_invoice_task.py

```diff
@@ -110,9 +112,35 @@ def test_create_rs_invoice_single_transaction(session):
     fee_schedule = FeeScheduleModel.find_by_filing_type_and_corp_type('CP', 'OTANN')
     line = factory_payment_line_item(invoice.id, fee_schedule_id=fee_schedule.fee_schedule_id)
     line.save()
-    assert invoice.invoice_status_code == InvoiceStatus.APPROVED.value
 
-    CreateInvoiceTask.create_invoices()
+    invoice_failed_data = {
+        'invoice_number': '123',
+        'Transfer-Encoding': 'chunked',
+        'Content-Length': '11388',
+        'Content-Type': 'text/html',
+        'CAS-Returned-Messages': '[Trx Errors = [9999] Transaction not created'}
```

Comment:
This probably causes a linter error } should probably be on a new line to be consistent

Transfer-Encoding, Content-Length, CAS-ReturnedMessages are headers, shouldn't belong in the data